config/jupyterhub_config.py

```diff
@@ -89,8 +89,7 @@
 c.DockerSpawner.debug = True
 
 c.DockerSpawner.cpu_limit = 1
-# for some reason, enabling mem limit makes spawned containers not work
-#c.DockerSpawner.mem_limit = "512"
+c.DockerSpawner.mem_limit = lambda s: '8M' if s.user.name in c.GitHubOAuthenticator.blocked_users else "2G"
```

Comment:
@charles-cooper for blocked users the memory limit is 8MB which is the minimum for docker - and will crash on startup